### PR TITLE
Ccg rhel9rhel #18472 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/18472

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I also add ES 9, but I need your help and review!

**Comment by keichwa** [comment]
[suse_manager_client-configuration_guide-rh9.pdf](https://github.com/uyuni-project/uyuni-docs/files/9662922/suse_manager_client-configuration_guide-rh9.pdf)
[uyuni_ccc-rh9.pdf](https://github.com/uyuni-project/uyuni-docs/files/9662979/uyuni_ccc-rh9.pdf)

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 1]
```diff
@@ -74,6 +74,7 @@ The products you need for this procedure are:
 
 | {es} 8
 | RHEL or SLES ES or CentOS 8 Base and SUSE Linux Enterprise Server with Expanded Support 8 x86_64
+
```
Comment (srbarrios) [inline modules/client-configuration/pages/clients-sleses.adoc:77]: @keichwa I would say we need to add version 9 here too.

Comment (keichwa) [reply modules/client-configuration/pages/clients-sleses.adoc:77]: ok, sounds sensible!

**Pull request merged**